### PR TITLE
bpo-32970: Improve disassembly of the MAKE_FUNCTION instruction.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue32970

### CODE REVIEW

File: Lib/dis.py [hunk 1]
```diff
@@ -339,12 +348,15 @@ def _get_instructions_bytes(code, varnames=None, names=None, constants=None,
             elif op in hasfree:
                 argval, argrepr = _get_name_info(arg, cells)
             elif op == FORMAT_VALUE:
-                argval = ((None, str, repr, ascii)[arg & 0x3], bool(arg & 0x4))
-                argrepr = ('', 'str', 'repr', 'ascii')[arg & 0x3]
+                argval, argrepr = FORMAT_VALUE_CONVERTERS[arg & 0x3]
+                argval = (argval, bool(arg & 0x4))
```
Comment (ncoghlan) [inline Lib/dis.py:352]: This seems like an unrelated refactoring?

**Review by ncoghlan — COMMENTED** [review]
The `MAKE_FUNCTION` change looks good to me, but the `FORMAT_VALUE` changes seem unrelated.

File: Lib/dis.py (hunk 1 above)
Comment (serhiy-storchaka) [reply Lib/dis.py:352]: Initially I implemented the logic for MAKE_FUNCTION as a sequence of ifs. But later I seen that it can be implemented using a single constant. Since I added the MAKE_FUNCTION_FLAGS global I take the chance to move constants for FORMAT_VALUE.
